runtime/modules/isp_qemu.py

```diff
@@ -150,10 +149,13 @@ def qemuOptions(exe_path, run_dir, extra, runtime, use_validator=True, gdb_port=
 
     # Machine selection
     if "sel4" in runtime:
```

Comment:
Is there a more generic runtime identifier we should use instead, since Linux/BSD/etc... will also probably make use of this configuration?